sdk/openai/Azure.AI.OpenAI/src/Streaming/StreamingChoice.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+
+namespace Azure.AI.OpenAI.Custom
+{
+    public class StreamingChoice
```

Comment:
Since this and `StreamingCompletions` are intended to eventually be in the cadl and therefore be generated these should go in the Custom folder.

Reply:
Thanks--I was curious about doing that in advance or not. I moved `StreamingCompletions` and `StreamingChoice` into `Custom` and the remaining new files into a `Helpers` folder (as the former "Streaming" identity got weaker).